specification/sections/01-flag-evaluation.md

```diff
@@ -423,19 +425,23 @@ stateDiagram-v2
     READY --> NOT_READY:shutdown()
     STALE --> NOT_READY:shutdown()
     ERROR --> NOT_READY:shutdown()
-    READY --> CONTEXT_PENDING:::optional:setContext()
-    CONTEXT_PENDING --> READY:*
+    READY --> RECONCILING:::client:setContext()
+    RECONCILING:::client --> READY:*
 
-    classDef optional fill:#999
+    classDef client fill:#555
 ```
 
 \* transitions occurring when associated events are spontaneously emitted from the provider
 
-Note that only SDKs implementing the [static context (client-side) paradigm](../glossary.md#static-context-paradigm) define `CONTEXT_PENDING` to facilitate [context reconciliation](./02-providers.md#26-provider-context-reconciliation).
+<span style="color:#555">█</span> only defined in static-context (client-side) paradigm
+
+Note that only SDKs implementing the [static context (client-side) paradigm](../glossary.md#static-context-paradigm) define `RECONCILING` to facilitate [context reconciliation](./02-providers.md#26-provider-context-reconciliation).
```

Comment:
Should we use the note GitHub syntax? It's now supported in the docs as well.

Reply:
Ya I think that's a nice touch. Will do.

Reply:
done